pages/linux/scrot.md

```diff
@@ -0,0 +1,15 @@
+# scrot
+
+> Capture screenshots using the command line.
+
+- Capture the whole screen:
+
+`scrot`
+
+- Select a window with the mouse then capture the window:
```

Comment:
Does this mean the capture waits until you click on a window with the mouse ? If yes, we should probably clarify that.